Flo February updates (#867)
akeneo/pim-helpcenter: content/updates/2021-02/4-Api-filter-on-product-quality-score.md

---
pim_announcement_audience:
- CE
- EE
---

# New API filter on product quality score
::: meta-data type="New" features="Connectivity" available="mid-February" link-to-doc="https://api.akeneo.com/documentation/filter.html#on-their-quality-score"

> When retrieving products through our Rest API, you can now filter them according to their quality score. Very useful if you want to distribute only highly rated products on your channels or if you want to select products whose quality needs to be improved.

Author: ```suggestion
When retrieving products through our REST API, you can now filter them according to their quality score. Very useful if you want to distribute only highly rated products on your channels or if you want to select products whose quality needs to be improved.
```

---
pim_announcement_audience:
- CE
- EE
---

# New API filter on product quality score
::: meta-data type="New" features="Connectivity" available="mid-February" link-to-doc="https://api.akeneo.com/documentation/filter.html#on-their-quality-score"

> When retrieving products through our Rest API, you can now filter them according to their quality score. Pretty useful when:

Reviewer: ```suggestion
When retrieving products through our Rest API, you can now filter them according to their quality score. Very useful if you want to distribute only highly rated products on your channels or if you want to select products whose quality needs to be improved.
```

> - you only want highly graded products to be distributed on your channels,

Reviewer: ```suggestion
```

> - or you want to select products on which proper enrichment work is still needed.

Reviewer: ```suggestion
```

Reviewer: removed the list to improve readability on the PIM.
